### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by nikolaitarasov** [zhuhaow/NEKit#284]
Currently I'm using HTTPProxySocket to read HTTP header information. But I need to get exact source app name which is not displayed in the header of course. Example, I need to see if Youtube or other app is launched. Is there any way to get apps names in NEKit framework?

**Comment by zhuhaow** [comment]
No. And I’m not sure if it’s possible to do it on iOS. It’s possible to do it as what lsof does, but I don’t know if that api is available on iOS.

> 在 2020年1月24日，12:07，nikolaitarasov <notifications@github.com> 写道：
> 

﻿
Currently I'm using HTTPProxySocket to read HTTP header information. But I need to get exact source app name which is not displayed in the header of course. Example, I need to see if Youtube or other app is launched. Is there any way to get apps names in NEKit framework?
